### PR TITLE
Adding Style Variation Sherbet #104

### LINKED ISSUE #104: Variation "Sherbet"
**Screenshot:**

Sherbet

**Figma:**
https://www.figma.com/file/p4qBWBvxIEvT0bAHdCwf23/TT3-Variation-%22Sherbet%22?node-id=2%3A9

**Theme.json:**
[variation-sherbet.json](https://github.com/WordPress/twentytwentythree/files/9471273/variation-sherbet.json.zip)

### PR DESCRIPTION
**Pull request by Dabalina**
This adds the Sherbet style variation with suggested changes. There is a suggestion to add a duotone like the one in retroactive, I went ahead and added the same duotone from retroactive, wondering if it should be a bit different but not sure what to do with it. 

Fixes #104

### CODE REVIEW

**Comment by scruffian** [comment]
If we want to the duotone to work in the same way as Retroactive we'd need to explicitly set the filters for the each of the blocks that it would apply to.

**Comment by mikachan** [comment]
Thanks for working on this!  ❤️

I've opened a [PR](https://github.com/WordPress/twentytwentythree/pull/171) to move the Inter font definition to the base theme, so we don't need to include the font in this variation.

File: styles/variation-sherbet.json [hunk 1]
```diff
@@ -0,0 +1,248 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 2,
+	"title": "Variation Sherbet",
```
Comment (mikachan) [inline styles/variation-sherbet.json:4]: The title should just be, "Sherbet".

File: styles/variation-sherbet.json [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+				},
+				{
+					"color": "#CCFFFF",
+					"name": "Tertiary",
+					"slug": "tertiary"
+				}
+			]
+		},
+		"typography": {
+			"fontFamilies": [
```
Comment (mikachan) [inline styles/variation-sherbet.json:55]: We don't need to redefine the font families in the variation file, so we can remove the 'fontFamilies' array.

**Review by mikachan — COMMENTED** [review]
Looking good! I've just left two small comments.

**Comment by mikachan** [comment]
In order to include this in the Beta 1 release, I merged these changes over in another PR https://github.com/WordPress/twentytwentythree/pull/174. @Dabalina, I added you as a co-author, and please feel free to open any further PRs with additional changes (to this variation or anything else).

**Pull request closed without merging**